src/hud_elements.cpp

```diff
@@ -105,14 +105,20 @@ void HudElements::version(){
 }
 
 void HudElements::gpu_stats(){
-    if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gpu_stats]){
+    for(auto per_gpu_info: gpu_info){
+         if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gpu_stats]){
         ImGui::TableNextRow(); ImGui::TableNextColumn();
-        const char* gpu_text;
```

Comment:
Any reason for turning it into std::string instead?

Reply:
I was concatenating the name of the GPU. I've since put that on a separate hud element but we could keep it as a `std::string` in case we need to add GPU indices (GPU 1,GPU 2 etc)